ci-operator/config/CSPI-QE/oadp-qe-automation/CSPI-QE-oadp-qe-automation-main.yaml

```diff
@@ -0,0 +1,107 @@
+      ENV GOROOT=/usr/local/go
+      ENV PATH=/usr/local/go/bin:$PATH
+      ENV GOBIN=/usr/local/go/bin
+      ENV GOCACHE=/tmp/
+      ENV BUCKET=interopoadp
+      ARG GO_VER=go1.18.10.linux-amd64.tar.gz
+      RUN yum update -y && yum install gcc -y && yum install diffutils -y
+      RUN curl -s -k https://dl.google.com/go/$GO_VER -o $GO_VER && \
+      tar -C /usr/local -xzf $GO_VER && rm -fr $GO_VER
+      RUN go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.6 && go install github.com/onsi/gomega/... && \
+      which ginkgo && \
+      chmod 777 -R $HOME/.cache && \
+      mkdir /go && chmod 777 /go && mkdir -p /alabama/.ansible/tmp && chmod 777 /alabama/.ansible/tmp
+      RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+          chmod 700 get_helm.sh && \
+          ./get_helm.sh
+  to: go-runner
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: src-oc
+- dockerfile_literal: |
+    FROM src-oc
+    RUN curl -s "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+      unzip -q awscliv2.zip && \
+      ./aws/install && \
+      rm -rf awscliv2.zip aws &&\
+      mkdir /.aws && chmod 777 /.aws
+  from: src-oc
+  to: aws-console
+- dockerfile_literal: |
+    FROM src-oc
+    RUN yum install findutils gcc git hostname iputils jq libcurl-devel libxml2-devel make openssl openssl-devel passwd pip python3-devel qemu-img redhat-rpm-config rsync rubygems ruby-devel sqlite which zip -y
+  from: src-oc
+  to: src-test
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: oadp-private-scenario
+  interval: 960h
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      OPERATORS: |
+        [
+        {"name": "redhat-oadp-operator", "source": "redhat-operators", "channel": "stable-1.1", "install_namespace": "openshift-adp", "target_namespaces": "openshift-adp", "operator_group":""},
+        {"name": "volsync-product", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group":"global-operators"},
+        {"name": "odf-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-storage", "target_namespaces": "openshift-storage", "operator_group":"openshift-storage"}
+        ]
+      BUCKET_NAME: oadptest
+      SPOT_INSTANCES: "true"
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    - ref: install-operators
+    - ref: lp-interop-tooling-create-s3
+    test:
+    - ref: interop-oadp-execute
+    post:
+    - ref: gather-aws-console
+    - ref: lp-interop-tooling-delete-s3
```

Comment:
Have you seen a situation where this post ref has failed and it prevents the chain: ipi-deprovision from being called?

I'm curious if we need to add something like [allow_best_effort_post_steps](https://steps.ci.openshift.org/ci-operator-reference#line456) to this or not.

Reply:
I have seen the post ref fail, but it has not prevented deprovision from being called.